admin/css/picker.css

```diff
@@ -82,6 +87,62 @@
   opacity: 100%;
 }
 
+.image-picker.open-sidebar .help {
+  display: none;
```

Comment:
That's necessary to hide it on mobile when the sidebar is open, because we have `z-index: 1`... However I changed the implementation a bit since and it seems that this `z-index` value is no longer needed, so I will update it!